1/README.md

```diff
@@ -21,16 +22,36 @@ This Specification is a [free and open standard](http://www.digistan.org/open-st
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
 
 ## Tango Specification
-The TANGO control system is a device-oriented controls toolkit for controlling any kind of hardware or software and building SCADA systems.
-
+The TANGO control system is a toolkit for building distributed control systems which are device oriented. 
```

Comment:
Likely, this going to be the very first section one would read when coming to Tango RFC... suggest to explain a little bit the concept "device oriented", e.g. "... which are device oriented, modelling the hardware equipment or the software component into the device abstraction". I think this can be improved how it is stated, though.